tests/test_integrate.py

```diff
@@ -153,6 +153,10 @@ def quantities_Haldane():
 def quantities_GaAs():
     return  ["berry_dipole","berry_dipole_test","berry_dipole_fsurf"]
 
+@pytest.fixture(scope="module")
+def quantities_sym_GaAs():
+    return  ["ahc"]
+
```

Comment:
Delete this block. No need to use fixture.

```suggestion
```

Reply:
Done